legion/legion/template_plugins/io.py

```diff
@@ -70,12 +71,30 @@
     watcher.watch(path=filepath, flags=aionotify.Flags.MODIFY)
 
     LOGGER.debug('Awaiting setup of watcher for file {}'.format(filepath))
-    await watcher.setup(template_system._loop)
+    event_loop = asyncio.new_event_loop()
+    asyncio.set_event_loop(event_loop)
+    future = asyncio.ensure_future(loop(event_loop, watcher, template_system, filepath, is_yaml, var_name))
+    event_loop.run_until_complete(future)
 
+
+async def loop(event_loop, watcher, template_system, filepath, is_yaml=False, var_name='file'):
+    """
+    Run aionotify watcher in current event loop
+    :param event_loop: Current event loop
```

Comment:
Pls add space before args

Reply:
The problem was resolved by this commit fc7ff0219e338fd298700b3e94636710810690c6